From: Sune Vuorela <debian@pusling.com>
Subject: Install headers where they were earlier
Forwarded: no

--- a/includes/CMakeLists.txt
+++ b/includes/CMakeLists.txt
@@ -87,7 +87,7 @@ install( FILES
   old/Phonon/VolumeFaderEffect
   old/Phonon/VolumeFaderInterface
   old/Phonon/VolumeSlider
-DESTINATION ${INCLUDE_INSTALL_DIR}/KDE/Phonon COMPONENT Devel)
+DESTINATION ${INCLUDE_INSTALL_DIR}/phonon/Phonon COMPONENT Devel)
 
 install(FILES
   old/Phonon/Experimental/AbstractVideoDataOutput
