From: Sune Vuorela <debian@pusling.com>
Subject: Install headers where they were earlier
Forwarded: no

Index: phonon/PhononConfig.cmake.in
===================================================================
--- phonon.orig/PhononConfig.cmake.in	2015-08-08 17:24:02.720798332 +0200
+++ phonon/PhononConfig.cmake.in	2015-08-08 17:25:36.869091094 +0200
@@ -34,7 +34,7 @@
 set(PHONON_INCLUDES ${PHONON_INCLUDE_DIR} ${PHONON_INCLUDE_DIR}/KDE)
 
 if (PHONON_QT_COMPAT_HEADERS)
-    list(APPEND PHONON_INCLUDES ${PHONON_INCLUDE_DIR}/Phonon)
+    list(APPEND PHONON_INCLUDES ${PHONON_INCLUDE_DIR}/phonon)
 endif()
 
 # Find Internal is included in the backends' finders rather than here.
