From 974fb5f6ef5e21d5e91df48025f4f7cfb857efc7 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Tue, 5 Apr 2011 10:56:14 +0200
Subject: [PATCH] use PHONON_EXPORT_DEPRECATED instead of PHONON_DEPRECATED+PHONON_EXPORT

---
 phonon/objectdescription.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/phonon/objectdescription.h b/phonon/objectdescription.h
index 51158fe..b18272c 100644
--- a/phonon/objectdescription.h
+++ b/phonon/objectdescription.h
@@ -346,7 +346,7 @@ typedef QPair<QByteArray, QString> DeviceAccess;
  */
 typedef QList<DeviceAccess> DeviceAccessList;
 
-void PHONON_DEPRECATED PHONON_EXPORT registerMetaTypes();
+void PHONON_EXPORT_DEPRECATED registerMetaTypes();
 
 } //namespace Phonon
 
-- 
1.7.4.1

